### ISSUE TITLE
Equalizer

### ISSUE THREAD

**Issue by griimtoclean** [filoe/cscore#158]
Setting the gain on any filter makes it sound like garbage.

SetGain is not working

**Comment by filoe** [comment]
Sorry for the late response.
To be honest it's hard to follow your explanation. 
The equalizer gets used within some samples and it does not sound like garbage at all.
It's also used in some projects like https://github.com/digimezzo/Dopamine. There was never any issue of any user that it sounds like garbage. 
So you may provide some code which reproduces your problem.
Otherwise I'm afraid that I can't help you.
